apiserver/allfacades.go

```diff
@@ -147,6 +147,7 @@ func AllFacades() *facade.Registry {
 	reg("Cleaner", 2, cleaner.NewCleanerAPI)
 	reg("Client", 1, client.NewFacade)
 	reg("Cloud", 1, cloud.NewFacade)
+	reg("Cloud", 3, cloud.NewFacadeV3) // adds CredentialContents
```

Comment:
Let's just keep facade as V2 and remove feature flag restriction. Only V1 is released. Removing flag means someone *could* script a client call to invoke a caas cloud api, but 1. they won't, 2. if they do, it won't get them very far

Reply:
ok